marklogic-client-api/src/main/java/com/marklogic/client/DatabaseClientFactory.java

```diff
@@ -973,13 +974,42 @@ public CertificateAuthContext(SSLContext context, SSLHostnameVerifier verifier,
      *           UnrecoverableKeyException.
      */
     public CertificateAuthContext(String certFile, X509TrustManager trustManager)
-        throws CertificateException, IOException,
-        UnrecoverableKeyException, KeyManagementException {
+        	throws CertificateException, IOException, UnrecoverableKeyException, KeyManagementException {
```

Comment:
Have this constructor call the next one, passing in TLSv1.2 as the value of protocol. Same goes for the other pair of constructors.